### ISSUE TITLE
Can't find police lockers

### ISSUE THREAD

**Issue by joaodutra11** [xabier1989/WiredPlayers-RP#32]
I tried to use /equipment command at the police lockers area and it's not working.

**Comment by AteshM** [comment]
Do you get any error? You should get a message!

**Comment by joaodutra11** [comment]
Yes, I get a error message saying that I'm not at the police lockers, but I am.
